src/Fmt_ast.ml

```diff
@@ -1479,6 +1479,7 @@ and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext
                             | Pexp_sequence _ ) } as e1 )
                     , _ ) } as str ) ] ) ->
       fmt_expression c ~box ?eol ~parens ~ext (sub_exp ~ctx:(Str str) e1)
+      $ fmt_atrs
```

Comment:
I have not tested, but I suspect that the whole body of this case would be better wrapped in `hvbox 0 ( ... )`.

Reply:
Added!